src/OrleansProviders/Streams/Common/CommonStreamProviderSettings.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Orleans.Providers.Streams.Common
+{
+    /// <summary>
```

Comment:
These settings are common only to the recoverable stream providers, not azure queue (a non-recoverable persistent stream provider) and sms (an ephemeral stream provider).  Maybe RecoverableStreamProviderSettings?